sky/cli.py

```diff
@@ -3062,13 +3063,20 @@ def _output():
                        '(including non-common ones) and pricing.')
                 return
 
+        new_gpu_name = gpu_name
```

Comment:
We should rename `gpu_name` to `accelerator_str` and update this method elsewhere (including docs) because

1. It can include TPUs and other accelerators
2. It can be `name` or `name:quantity`